snow/networking/router/chain_router_test.go

```diff
@@ -1526,7 +1578,9 @@ func TestValidatorOnlyAllowedNodeMessageDrops(t *testing.T) {
 		prometheus.NewRegistry(),
 	)
 	require.NoError(err)
+
```

Comment:
nit: clear out these new line diffs

Reply:
@joshua-kim , If you see above I grouped
```
go tm.Dispatch()
defer tm.Stop()
```
in all tests. Should I revert the lines in all tests?

Reply:
Ah I see